tests/integration/test_elasticsearch.py

```diff
@@ -211,29 +274,35 @@ def _create_domain(cls, name=None, version=None, es_cluster_config=None):
         # wait for completion status
         def check_cluster_ready(*args):
             status = es_client.describe_elasticsearch_domain(DomainName=name)
-            created = status["DomainStatus"]["Created"]
-            LOG.info("asserting created state of domain %s (state = %s)", name, created)
-            assert created, "gave up waiting on cluster to be ready"
+            processing = status["DomainStatus"]["Processing"]
+            LOG.info(
+                "asserting that cluster of domain %s is not processing (processing = %s)",
+                name,
+                processing,
+            )
+            assert processing is False, "gave up waiting on cluster to be ready"
+
+            # also check that the cluster is healthy
+            try_cluster_health(f"http://{status['DomainStatus']['Endpoint']}")
 
-        retry(check_cluster_ready, sleep=10, retries=12)
+        retry(check_cluster_ready, sleep=10, retries=24)
 
 
-class TestCustomEndpointElasticsearchCluster:
+class TestEdgeProxiedElasticsearchCluster:
     def test_route_through_edge(self):
-        cluster = CustomEndpointElasticsearchCluster(
-            CustomEndpoint(endpoint="http://localhost:4566/my-es-cluster", enabled=True)
-        )
+        cluster_id = f"domain-{short_uid()}"
+        cluster_url = f"http://localhost:4566/{cluster_id}"
```

Comment:
nit: we could insert `config.EDGE_PORT` here, in case we ever want to run the tests with a different edge port.